app/src/main/java/org/covidwatch/android/di/AppContext.kt

```diff
@@ -0,0 +1,8 @@
+package org.covidwatch.android.di
+
+import android.content.Context
+import android.content.ContextWrapper
+
+// exists solely to make the Android applicationContext injectable
+class AppContext(context : Context) : ContextWrapper(context) {
```

Comment:
Remove it please since we won't use the implementation of `NTPTime` with `companion object`.

Reply:
file removed